packages/i18n-utils/src/test/localize-url.js

```diff
@@ -91,12 +91,35 @@ describe( '#localizeUrl', () => {
 	} );
 
 	test( 'trailing slash variations', () => {
-		expect( localizeUrl( 'https://automattic.com/cookies/', 'de' ) ).toEqual(
+		const loggedIn = false;
```

Comment:
Should this be renamed to reflect the actual arg name?

Reply:
Yup, I can definitely do that 👍